scripts/servers.sh

```diff
@@ -405,14 +412,6 @@ function passthetorch() {
     exit 1
   fi
 
-  server_status=`echo "$curl_result" | tail -1`
-  if [ "${server_status}" != "200" ]; then
```

Comment:
I don't feel comfortable skipping an entire sanity check just because we don't like the way it behaves.
We should ideally change the behavior of the underlying check (a) not to return 50X when there are bad Stripe charges (b) change the check to accept 50X if the HTML body contains the word "Stripe" (c) actually care about the specific Stripe charges that are currently causing the error.

Reply:
Ok, I put it back in